src/entity-parser.ts

```diff
@@ -3,7 +3,11 @@ const CHARCODE    = /^#([0-9]+)$/;
 const NAMED       = /^([A-Za-z0-9]+)$/;
 
 export default class EntityParser {
-  constructor(private named) {}
+  private named: {};
+
+  constructor(named) {
```

Comment:
If you leave it the way it was, this is the default.

So basically:

```ts
class Foo {
  constructor(private named) {}
}
```

And this:

```ts
class Foo {
  private named: {};

  constructor(named) {
    this.named = named;
  }
}
```

Are the same, but the first version is much shorter and has less repetition...

See [output in the playground](http://www.typescriptlang.org/play/index.html#src=class%20Foo%20%7B%0D%0A%20%20constructor(private%20named)%20%7B%7D%0D%0A%7D).